data/lua/wml/message.lua

```diff
@@ -282,12 +281,12 @@ function wesnoth.wml_actions.message(cfg)
 
 	-- Only the first text_input tag is considered
 	local text_input
-	for cfg in helper.child_range(cfg, "text_input") do
+	for cfg_text in helper.child_range(cfg, "text_input") do
```

Comment:
Minor, but I think I prefer reversing this (ie `text_cfg` rather than `cfg_text`).

Reply:
This isn't 1974 and we're not limited to 8-character variable names (6 if you're on a PDP). Use more descriptive names to aid the reader later on.

`text_input_configuration` would be my suggestion. But if you insist on using cfg, `text_input_cfg`

Reply:
Sure, but I don't like having really long names, either.

Reply:
well `cfg` is a _very_ common abbreviation within wesnoth, so I'm not really in a mood to change `cfg` specifically. I'll change in other ways as you wrote tho.

Reply:
done. (Chose `text_input_cfg`.)